src/open_inwoner/templates/pages/product/detail.html

```diff
@@ -74,12 +74,12 @@ <h1 class="utrecht-heading-1" id="title">
             {% render_grid %}
                 {% render_column span=6 compact=True %}
                     <h3 class="h3">{% trans "U komt in aanmerking" %}</h3>
-                    {% condition_list conditions=object.conditions.all %}
+                    {% include "components/conditions/ConditionList.html" with conditions=object.conditions.all only}
```

Comment:
These are currently not rendering:

![image](https://github.com/maykinmedia/open-inwoner/assets/29249171/81720c78-6b04-42fe-8018-3f928f718ce7)

```suggestion
                    {% include "components/Condition/ConditionList.html" with conditions=object.conditions.all only %}
```

There probably is no test for this because CI passed, could you add one?